reloj-checador/app/javascript/stylesheets/application.scss

```diff
@@ -0,0 +1 @@
+@import url(devise.scss);
```

Comment:
I think you can import the scss file in another way, @Chavin30 is there another way?